base/task.jl

```diff
@@ -1097,6 +1118,12 @@ function try_yieldto(undo)
         rethrow()
     end
     ct = current_task()
+    # scheduler -task-started-> user
+    # scheduler -task-resumed-> user
+    if ct.is_timing_enabled
+        # @assert ct.last_dequeued_at == 0
```

Comment:
yeah, per my other comment, i think we can delete this, so that we don't need to waste the perf on zeroing it?